configure.cmd

```diff
@@ -24,7 +24,7 @@ set REACTOS_SOURCE_DIR=%~dp0
 REM Ensure there's no spaces in the source path
 echo %REACTOS_SOURCE_DIR%| find " " > NUL
 if %ERRORLEVEL% == 0 (
-    echo. && echo   Your source path contains at least one space.
+    echo. & echo   Your source path contains at least one space.
```

Comment:
No quotes around this string?
And what issue does replacing `&&` with `&` solve?

Reply:
That's not the .sh script, but .cmd. And in batch, the `echo` command doesn't interpret quotes around strings: it just outputs them verbatim.